web-chat/src/MessageInput.tsx

```diff
@@ -1,7 +1,15 @@
-import React, { ChangeEvent, KeyboardEvent, useState } from 'react';
-import { Button, Grid, TextField } from '@material-ui/core';
+import { ChangeEvent, KeyboardEvent, useState } from 'react';
 import { useWaku } from './WakuContext';
 
+import {
```

Comment:
Did the linter add this newline?
```suggestion
import {
```

Reply:
I'm guessing so, but it's possible VSCode did; I've been running "npm run fix" on the web-chat directory before I commit, but it should be automated with a pre-commit hook as you mentioned.